ui/src/conferences/components/ConferenceWebsitesAction.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { List } from 'immutable';
+import { Menu, Icon, Tooltip } from 'antd';
+
+import ExternalLink from '../../common/components/ExternalLink';
+import { removeProtocolAndWwwFromUrl } from '../../common/utils';
+import ActionsDropdownOrAction from '../../common/components/ActionsDropdownOrAction';
+
+function renderWebsitesDropdownAction(website) {
+  const href = website.get('value');
+  const display =
+    website.get('description') || removeProtocolAndWwwFromUrl(href);
+  return (
+    <Menu.Item key={href}>
+      <ExternalLink href={href}>{display}</ExternalLink>
+    </Menu.Item>
+  );
+}
+
+function renderWebsiteAction(website, title) {
+  return <ExternalLink href={website.get('value')}>{title}</ExternalLink>;
+}
+
+const ACTION_TITLE = (
+  <Tooltip title="Conference website">
```

Comment:
in the mockup it has the link icon and the text ```websites``` next to it and not a tooltip. A tooltip would make it more consistent with what is in authors, I agree. But should we ask Stella too just to make sure?

Reply:
We should have the text 'website' next to the icon to be consistent with all the literature search. We don't have tooltips in literature, but having it here is fine! We should consider adding tooltips in all buttons, as it's a way to help users.

Reply:
so you would need to add the text 'website' as well

Reply:
I do disagree with idea of putting tooltips in general... The most important reason is that they don't work on mobile. Especially that, we are not trying more intuitive and obvious options for UI, because well, we have tooltips for that. It's alright for the Desktop users but then mobile UX suffers, since there is no explanation for the vague feature/functionality on mobile.

Reply:
On another note, we have some sort of consistency with the actions... they have tooltip if they are only an icon, they don't when they have also text next the icon.